includes/class-vindi-payment.php

```diff
@@ -68,7 +68,15 @@ public function get_plan()
 
         foreach($items as $item) {
             $product    = $this->order->get_product_from_item($item);
-            $vindi_plan = get_post_meta($product->id, 'vindi_subscription_plan', true);
+
+            if( isset($item['variation_id']) && $item['variation_id'] != 0){
```

Comment:
```suggestion
            if (isset($item['variation_id']) && $item['variation_id'] != 0) {
```

podemos escrever assim?

Reply:
Sim